### PR TITLE
add section to the docs discussing source control tradeoffs

### LINKED ISSUE #446: Include .gitignore section in docs
The deployments folder should likely be included in your project's repo, to allow for using the successfully deployed contracts.

This will vary project to project, but the docs should include a section on the trade-offs (repo bloat vs access to all the information).

## TODO

- [ ] Determine where to add the section
- [ ] Consider whether we are giving fine grained advice (consequences of leaving out the journal for instance)
- [ ] Add the section

### PR DESCRIPTION
**Pull request by zoeyTM**
fixes #446

### CODE REVIEW

File: docs/running-a-deployment.md [hunk 1]
```diff
@@ -107,6 +108,12 @@ The value of `requiredConfirmations` is the number of blocks after a transaction
 
 ---
 
+## Source control for deployments
+
+Ignition creates several files when a deployment is run. You may want to commit some or all of these files to source control.
+
+While committing the entire `deployments` directory is the recommended approach, there are some reasons why you may want to commit only some of the files: namely, repo bloat. The `deployments` directory can grow quite large, especially if you are deploying to multiple networks. At the very least, you should commit the `deployed_addresses.json` file found within each deployment directory. This file contains the addresses of all contracts deployed by the module.
```
Comment (alcuadrado) [inline docs/running-a-deployment.md:115]: Can we add something like:

You should make sure to store the rest of the files if you want to resume a deployment later.

Future versions of Ignition will make the `deployments` file system structure lighter, and friendlier to versioning.

**Review by alcuadrado — APPROVED** [review]
Approved with a small comment

File: docs/running-a-deployment.md (hunk 1 above)
Comment (alcuadrado) [reply docs/running-a-deployment.md:115]: Feel free to use your own wording

Comment (zoeyTM) [reply docs/running-a-deployment.md:115]: Done!

**Pull request merged**